rpcs3/Input/pad_thread.cpp

```diff
@@ -603,6 +603,7 @@ void pad_thread::UnregisterLddPad(u32 handle)
 	ensure(handle < m_pads.size());
 
 	m_pads[handle]->ldd = false;
+	m_pads[handle]->m_port_status = CELL_PAD_STATUS_DISCONNECTED;
```

Comment:
It should probably be:
```suggestion
	m_pads[handle]->m_port_status &= ~CELL_PAD_STATUS_CONNECTED;
	m_pads[handle]->m_port_status |= CELL_PAD_STATUS_ASSIGN_CHANGES;
```

Reply:
What exactly does assign changes do?